python-interface/src/MiniBotFramework/Sensing/Sensor.py

```diff
@@ -2,6 +2,7 @@
 class Sensor:
     def __init__(self, bot, name):
         self.name = name
+        # print "Sensor init called! Name: " + name
```

Comment:
Okay, I've deleted the print in MiniBot.py's register_sensor(), and made a similar change for Actuator.py and MiniBot.py's register_actuator().